.github/workflows/stale.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@v4
+      - uses: actions/stale@v5
```

Comment:
Consider this approach https://github.com/open-telemetry/opentelemetry-cpp/pull/1323.